docs/pipelines.md

```diff
@@ -95,7 +95,7 @@ A `Pipeline` definition supports the following fields:
         a `Task` requires.
         - [`from`](#using-the-from-field) - Indicates the data for a [`PipelineResource`](resources.md)
           originates from the output of a previous `Task`.
-      - [`runAfter`](#using-the-runafter-field) - Indicates that a `Task` should execute after one or more other 
```

Comment:
nit: the changes in this file all seem to be about removing spaces. Can we revert them?

Reply:
sure, let me figure it out. It must be autoformat of my IDE.

Reply:
the main branch has been formatted, 😄